src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/ExpanderTests/Expander_Bindings_TemplatedControl.xaml.cs

```diff
@@ -0,0 +1,53 @@
+#nullable enable
+using Uno.UI.Samples.Controls;
+using Microsoft.UI.Xaml.Automation.Peers;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml;
+using Microsoft.Graph.Models;
+using System;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace UITests.Shared.Microsoft_UI_Xaml_Controls.ExpanderTests;
+
+[Sample("Expander", "MUX", Name = "Expander_Bindings_TemplatedControl", IsManualTest = true)]
```

Comment:
Provide some description on what the manual test should be using the `Description =` parameter